### ISSUE TITLE
Scanner(with debug cert) loosing sometimes from safari development

### ISSUE THREAD

**Issue by aromanyuk** [AppGyver/steroids#763]
If i run my app in Scanner(debugger) on iPhone i see my phone in Safari Development tab but when i inspect my app i need restart Safari time to time because i can't saw my phone in Safari Development tab after several minutes.

**Comment by pre** [comment]
Connection to the Safari dev tools is unstable at times. Usually reconnecting your device and/or restarting Safari will help.

**Comment by Ezku** [comment]
I experience consistent disconnects from debugger when trying to use the camera.

Repro steps:
1. `<input type="file" />`
2. Open page in Safari console, `setInterval(function() { console.log('still alive'); }, 1000)`
3. Press input field on device
4. `Choose Existing` => still alive
5. `Take Photo or Video` => Safari disconnects

**Comment by aromanyuk** [comment]
Yes, restarting Safari fix that, but only for few minutes. So currently I need restart Safari every 5-7 minutes and this is very uncomfortable.

**Comment by pre** [comment]
@aromanyuk what iOS version you have? Can you find any pattern, ie "if I do X, then device is disconnected" or such?
